cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -222,16 +223,25 @@ func getAuthzModes(authzModeExtraArgs string) string {
 
 		// only return the user provided mode if at least one was valid
 		if len(mode) > 0 {
-			klog.Warningf("the default kube-apiserver authorization-mode is %q; using %q",
-				strings.Join(defaultMode, ","),
-				strings.Join(mode, ","),
-			)
+			if !compareMode(defaultMode, mode) {
+				klog.Warningf("the default kube-apiserver authorization-mode is %q; using %q",
+					strings.Join(defaultMode, ","),
+					strings.Join(mode, ","),
+				)
+			}
 			return strings.Join(mode, ",")
 		}
 	}
 	return strings.Join(defaultMode, ",")
 }
 
+// compare two modes which contents are same regardless of index.
+func compareMode(modeA, modeB []string) bool {
+	sort.Strings(modeA)
+	sort.Strings(modeB)
+	return strings.Join(modeA, ",") == strings.Join(modeB, ",")
```

Comment:
Thanks for your PR! I concern about strings.Join() would be called three times:
1. The `compareMode` func
2. klog
3. func `getAuthzModes` return
I think it's better to optimize this.

Reply:
i've seen the warning myself and i acknowledge that this should be fixed.
but the ordering of auth modes for the API server is important, so there should be no sorting!

the compare function should look like this:
```
// compareAuthzModes compares two given authz modes and returns false if they do not match
func compareAuthzModes(a, b []string) bool {
	if len(a) != len(b) {
		return false
	}
	for i, m := range a {
		if m != b[i] {
			return false
		}
	}
	return true
}
```
@seongenie please update the PR to use that, and also use `git commit -a` and `git push -f` to keep the number of commits to 1.

thanks

Reply:
you can also use:
`./hack/verify-gofmt.sh` to make sure the code is formatted correctly.